### PR TITLE
Update dependencies and lint rules

### PR DESCRIPTION
**Pull request by Deivis**
This PR updates some important dependencies like `eslint`, `former-kit-skin-pagarme`, `eslint-config-pagarme-react` and `stylelint-config-pagarme-react` which should need some lint fixes in the code.
## Context

Update dependencies and apply new lint rules.
## Checklist
- [x] Update dependencies
- [x] Apply the new lint rules to the code

## Linkded issues
- [x] Unblock this https://github.com/pagarme/react-scripts-former-kit-dashboard/issues/29

### CODE REVIEW

File: src/CalendarInput/index.js [hunk 1]
```diff
@@ -172,14 +173,14 @@ class CalendarInput extends Component {
     const showDateSelector = !(
       !isPeriodSelection(dateSelection)
       && closeOnSelect
-      && this.state.showDateSelector
+      && this.state.showDateSelector // eslint-disable-line react/destructuring-assignment
```
Comment (rsmelo) [inline src/CalendarInput/index.js]: maybe you can use destructuring and rename here and you will not  need to use eslint-disable-line 
```suggestion
          const {
      showDateSelector: currentShowDateSelect,
    } = this.state
    const showDateSelector = !(
      !isPeriodSelection(dateSelection)
      && closeOnSelect
      && currentShowDateSelect
    )
```

File: src/Card/CardTitle.js [hunk 2]
```diff
@@ -49,36 +49,22 @@ export const CardTitle = ({
   return (
     <div {...getProps(onClick)}>
       {
-        (typeof title === 'string') &&
-          <h2>
-            {icon}
-            {title}
-          </h2>
+        (typeof title === 'string') && <h2> {icon} {title} </h2>
```
Comment (rsmelo) [inline src/Card/CardTitle.js]: Maybe you can add `(` and break the line? I don't know it looks difficult to read in a single line
```suggestion
(typeof title === 'string') && (
  <h2>{icon}{title}</h2>
)
```

File: src/Card/CardTitle.js [hunk 3]
```diff
@@ -49,36 +49,22 @@ export const CardTitle = ({
...
       }
       {
-        (typeof title !== 'string') &&
-          title
+        (typeof title !== 'string') && title
       }
       {
-        (typeof subtitle === 'string') &&
-          <h3>{subtitle}</h3>
+        (typeof subtitle === 'string') && <h3>{subtitle}</h3>
```
Comment (rsmelo) [inline src/Card/CardTitle.js]: Maybe you can add `(` and break the line? I don't know it looks difficult to read in a single line

File: src/Checkbox/Checkbox.js [hunk 4]
```diff
@@ -52,27 +52,12 @@ const Checkbox = ({
         </span>
       </label>
 
-      {error &&
-        <p className={theme.secondaryText}>
-          {error}
-        </p>
-      }
+      {error && <p className={theme.secondaryText}>{error}</p>}
```
Comment (rsmelo) [inline src/Checkbox/Checkbox.js]: Maybe you can add `(` and break the line? I don't know it looks difficult to read in a single line

File: src/CheckboxGroup/index.js [hunk 5]
```diff
@@ -74,55 +74,41 @@ class CheckboxGroup extends React.Component {
...
       <div className={rootClassName}>
         <div>
           {checkboxes}
         </div>
-        {error &&
-          <p className={secondaryTextClass}>
-            {error}
-          </p>
-        }
+        {error && <p className={secondaryTextClass}>{error}</p>}
```
Comment (rsmelo) [inline src/CheckboxGroup/index.js]: Maybe you can add `(` and break the line? I don't know it looks difficult to read in a single line

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbb6f21e6d9ff2e96ac4ced9d06554cd9232de24
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbb6f21e6d9ff2e96ac4ced9d06554cd9232de24
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: src/Table/TableHead.js [hunk 6]
```diff
@@ -92,22 +92,22 @@ class TableHead extends Component {
...
           </div>
         </th>
       )
     }
 
-    const thProps = disabled || not(onOrderChange) ? {} :
-      {
-        onClick: () => this.handleOrderChange(index),
-      }
+    const thProps = disabled
```
Comment (rsmelo) [inline src/Table/TableHead.js]:
```suggestion
    const thProps = disabled || not(onOrderChange)
```
maybe is better to read if there is no line break on conditional

File: .storybook/readmeDecorator.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+configureReadme({
+  StoryPreview: ({ children }) => <Fragment>{children}</Fragment>,
+})
+
+addParameters({
+  readme: {
+    codeTheme: 'darcula',
+  },
+});
```
Comment (miguelssrs) [inline .storybook/readmeDecorator.js]: precisa desse `;` aqui?

Comment (Deivis) [reply .storybook/readmeDecorator.js]: nope

File: src/Table/tests/Table.static.test.js [hunk 8]
```diff
@@ -136,15 +137,16 @@ describe('Table', () => {
...
-            .findWhere(node =>
-              node.length > 0 && node.type() === 'div' &&
-              node.find('span').length > 0 &&
-              node.find('input').lenght <= 0
-            )
+            .findWhere(node => (
+              node.length > 0
+              && node.type() === 'div'
+              && node.find('span').length > 0
+              && node.find('input').lenght <= 0
```
Comment (rsmelo) [inline src/Table/tests/Table.static.test.js]: typo, but it already exists in the previous code 
```suggestion
  && node.find('input').length <= 0
```

**Pull request merged**